tests/test_srt_reader.py

```
@@ -38,7 +38,7 @@ def test_read_srt_valid_file(temp_srt_file, valid_srt_content):
 
 def test_read_srt_file_not_found():
     """Test handling of non-existent file"""
-    with pytest.raises(SystemExit):
+    with pytest.raises(ValueError):
```

Ruff (0.8.2)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)